massa-graph/src/block_graph.rs

```diff
@@ -3122,13 +3196,26 @@ impl BlockGraph {
         for (thread, id) in latest_final_blocks.iter().enumerate() {
             let mut current_block_id = *id;
             while let Some(current_block) = self.get_active_block(&current_block_id) {
+                let parent_id = {
```

Comment:
isn't the parent info available in current_block ?

Reply:
Yes, retrieve removed.